src/devices/wholeBodyDynamics/app/wholebodydynamics-icub-external-six-fts-sim.xml

```diff
@@ -34,6 +38,8 @@
         </group>
 
         <group name="WBD_OUTPUT_EXTERNAL_WRENCH_PORTS">
+            <param name="/wholeBodyDynamics/left_arm/endEffectorWrench:o">(l_hand,l_hand_dh_frame,l_sole)</param>
```

Comment:
Why are you expressing this force with the orientation of the `l_sole` ? This configuration file is supposed to be general, i.e. working in a meaningful way even when the `l_sole` frame is not attached to the ground.

Reply:
I copy-pasted ports definitions from `iCubGenova02` file (wholeBodyDynamics.xml`). Should I just remove `l_sole` and `r_sole`?